docs/source/dev/imageseries-usage.rst

```diff
@@ -0,0 +1,58 @@
+.. _examples:
+
+Example of Imageseries Usage
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Here is an example of how the imageseries is used. One thing we commonly do is
+to process the raw image files by adding flips and subtract off the background.
+Then we save it as a new imageseries. This example saves it into a HDF5 file,
+but it is more common to use the frame-cache (sparse matrices), which is way
+smaller.
+
+.. code-block:: python
+
+    import numpy as np
+
+    from hexrd import imageseries
+    from hexrd.imageseries.omega import OmegaWedges
+
+    # Inputs
+    DarkFile = 'dark-50pct-100f'
+    H5File = 'example.h5'
```

Comment:
Python convention is to only use `CamelCase` for classes, not for variable names, and use `snake_case` for variable names instead.